build.proj

```diff
@@ -3,8 +3,11 @@
   <PropertyGroup>
     <!-- Capture OSGroup passed to command line for setting default FilterToOSGroup value below -->
     <_OriginalOSGroup>$(OSGroup)</_OriginalOSGroup>
-    <TestOsTarget Condition="'$(OSGroup)'!=''">$(OSGroup)</TestOsTarget>
-    <TestOsTarget Condition="'$(OSGroup)'==''">$(OSEnvironment)</TestOsTarget>
+  </PropertyGroup>
+  <PropertyGroup>
+    <InputOSGroup>$(OSGroup)</InputOSGroup>
```

Comment:
InputOSGroup - Are you trying to use this to capture what is passed at the command line?